### PR TITLE
Implement deliverables autosort & Create more test cases

### LINKED ISSUE #62: Add testing for deliverable fields, commands and parsing

### LINKED ISSUE #81: As a product manager, I can view my deliverables as a sorted whole
Sorted by deadline

### PR DESCRIPTION
**Pull request by gabztcr**
Test cases for list, find and clear commands for deliverables. Fixes #81 
Fix minor typos and wording for some JavaDocs. (Post-mortemly) fixes #62

### CODE REVIEW

File: src/main/java/seedu/address/model/deliverable/deliverable/UniqueDeliverableList.java [hunk 1]
```diff
@@ -72,6 +77,7 @@ public void remove(Deliverable toRemove) {
     public void setDeliverables(UniqueDeliverableList replacement) {
         requireNonNull(replacement);
         internalList.setAll(replacement.internalList);
+        sortList();
```
Comment (chrystalquek) [inline src/main/java/seedu/address/model/deliverable/deliverable/UniqueDeliverableList.java:80]: not necessary

File: src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import seedu.address.model.UserPrefs;
+import seedu.address.model.deliverable.DeliverableBook;
+import seedu.address.model.deliverable.ModelDeliverable;
+import seedu.address.model.deliverable.ModelDeliverableManager;
+
+public class ClearCommandTest {
+
+    @Test
+    public void execute_emptyAddressBook_success() {
```
Comment (chrystalquek) [inline src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java:17]: rename to correct test names and variable names

File: src/main/java/seedu/address/model/person/util/PersonComparator.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
  */
 public class PersonComparator implements Comparator<Person> {
     /**
-     * Compares Persons by Name in ascending order.
+     * Compares Persons by Name in alphabetical order.
```
Comment (chrystalquek) [inline src/main/java/seedu/address/model/person/util/PersonComparator.java:15]: thanks for updating the javadocs

Comment (gabztcr) [reply src/main/java/seedu/address/model/person/util/PersonComparator.java:15]: 🦆

File: src/main/java/seedu/address/model/deliverable/util/TimeEventComparator.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.time.LocalDateTime;
+import java.util.Comparator;
+
+import seedu.address.model.event.TimeEvent;
+
+/**
+ * Represents a Comparator for Deliverable which sorts by Deadline.
+ */
+public class TimeEventComparator implements Comparator<TimeEvent> {
```
Comment (chrystalquek) [inline src/main/java/seedu/address/model/deliverable/util/TimeEventComparator.java:11]: Move src/main/java/seedu/address/model/meeting/util/TimeEventComparator.java to src/main/java/seedu/address/model/util/TimeEventComparator.java and use the same class

File: src/main/java/seedu/address/model/meeting/meeting/Meeting.java [hunk 5]
```diff
@@ -67,6 +67,10 @@ public From getFrom() {
         return from;
     }
 
+    public LocalDateTime getFromLocalDateTime() {
```
Comment (chrystalquek) [inline src/main/java/seedu/address/model/meeting/meeting/Meeting.java:70]: remove

File: src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import seedu.address.model.deliverable.DeliverableBook;
+import seedu.address.model.deliverable.ModelDeliverable;
+import seedu.address.model.deliverable.ModelDeliverableManager;
+
+public class ClearCommandTest {
+
+    @Test
+    public void execute_emptyDeliverableBook_success() {
+        ModelDeliverable modelDeliverable = new ModelDeliverableManager();
+        ModelDeliverable expectedModelPerson = new ModelDeliverableManager();
```
Comment (chrystalquek) [inline src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java:19]: update variable name

File: src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+        assertCommandSuccess(new ClearCommand(), modelDeliverable, ClearCommand.MESSAGE_SUCCESS, expectedModelPerson);
+    }
+
+    @Test
+    public void execute_nonEmptyDeliverableBook_success() {
+        ModelDeliverable modelPerson = new ModelDeliverableManager(getTypicalDeliverableBook(), new UserPrefs());
```
Comment (chrystalquek) [inline src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java:26]: update variable names

**Review by chrystalquek — APPROVED** [review]
LGTM

**Pull request merged**